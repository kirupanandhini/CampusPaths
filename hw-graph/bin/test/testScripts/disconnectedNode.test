# Checks graphs with three nodes and one edge that only connects two of
# the nodes

# Create a graph
CreateGraph graph1

# Add three nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add an edge
AddEdge graph1 n1 n2 e1

# Print the nodes in the graph and the outgoing edges from n1, n2, and n3
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2
ListChildren graph1 n3