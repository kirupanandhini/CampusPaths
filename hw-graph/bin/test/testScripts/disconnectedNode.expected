# Checks graphs with three nodes and one edge that only connects two of
# the nodes

# Create a graph
created graph graph1

# Add three nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add an edge
added edge e1 from n1 to n2 in graph1

# Print the nodes in the graph and the outgoing edges from n1, n2, and n3
graph1 contains: n1 n2 n3
the children of n1 in graph1 are: n2(e1)
the children of n2 in graph1 are:
the children of n3 in graph1 are: